snapcraft/internal/sources/_subversion.py

```diff
@@ -81,8 +81,11 @@ def _get_source_details(self):
         commit = self.source_commit
 
         if not commit:  # retrieve the commit id
-            lines = subprocess.check_output(['svn', 'info', self.source_dir]
-                                            ).decode('utf-8').split('\n')
+            env = os.environ.copy()
+            env['LANG'] = 'C'
+            lines = subprocess.check_output(
+                ['svn', 'info', self.source_dir],
+                env=env).decode('utf-8').split('\n')
             prefix = 'Last Changed Rev: '
```

Comment:
ugh, this is so flaky :(
I searched for a better way to do this and found https://stackoverflow.com/a/44632181/2665322
`svn info --show-item revision --no-newline` seems to do the same. @sergiusens @kyrofa any memories of why we are doing this weird thing here?

Reply:
@elopio interesting!
Shouldn't it be `svn info --show-item last-changed-revision --no-newline`, instead?

Reply:
makes sense to me.

Reply:
alright, so this PR doesn't have sense anymore, I think.
If you all agree, I'd propose a new one that uses the above command in place of parsing `svn info` output. This way we'll remove any dependency from the language and fix the bug as well.